src/sentry/integrations/slack/webhooks/event.py

```diff
@@ -179,6 +179,11 @@ def on_link_shared(self, request: Request, slack_request: SlackDMRequest) -> boo
         if not results:
             return False
 
+        if features.has("organizations:slack-block-kit", organization):
+            for link_info in results.values():
+                if "text" in link_info:
+                    del link_info["text"]
```

Comment:
@isabellaenriquez a comment explaining why we have to delete this field would be nice